src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -131,8 +131,8 @@ export function createMessageContent(
     if (replyToEvent) {
         addReplyToMessageContent(content, replyToEvent, {
             permalinkCreator,
-            includeLegacyFallback: true,
-            renderIn: getRenderInMixin(relation),
+            includeLegacyFallback: includeReplyLegacyFallback,
```

Comment:
Weirdly this prop/param only ever gets set to false in tests, yet that specific test isn't testing this.